cbg_equation.py

```diff
@@ -1,9 +1,6 @@
 import numpy as np 
-import matplotlib.pyplot as plt 
 from scipy.integrate import odeint 
 import random
-from scipy import interpolate
-import statsmodels.api as sm
 
 def simulator(initial_carbs=121.7, initial_sugar=90, digestion_rate=0.0453, insulin_rate=0.0224, minutes=100, start_time=0):
```

Comment:
These are some oddly specific default params. Maybe add a comment, at least stating the obvious, which I assume is that these are the defaults because they seem to work well as such after your fiddling and trial & error?